### PR TITLE
chore: setup percy

### PR DESCRIPTION
**Pull request by andreapiras**
## 🖼 Context

This PR replaces Chromatic with Perci setup. Since Perci has authorized access to our repo we can setup a proper workflow for visual regression

## 🚀 Changes

 
Removed Chromatic, added Perci

## ✅ Checklist

- [ ] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -9,7 +8,7 @@ aliases:
   - &container
     working_directory: *workspace-root
     docker:
-      - image: circleci/node:lts
+      - image: circleci/node:lts-browsers
```
Comment (dozortsev) [inline .circleci/config.yml:11]: what this for?

File: .circleci/config.yml [hunk 2]
```diff
@@ -219,6 +225,14 @@ workflows:
                 - gh-pages
                 - main
 
+      - visual-test-main:
+          requires:
+            - test
+          filters:
+            branches:
+              only:
+                - main
```
Comment (dozortsev) [inline .circleci/config.yml:234]: can you explain what this job for? against which branch Percy would compare results on `main` branch?

File: packages/components/package.json [hunk 3]
```diff
@@ -39,11 +39,10 @@
     "test": "jest --env=jsdom --maxWorkers=4 --config=../../node_modules/@transferwise/test-config/jest.config.js",
     "test:watch": "jest --watch --env=jsdom --config=../../node_modules/@transferwise/test-config/jest.config.js",
     "lint": "jest --maxWorkers=4 --config ../../node_modules/@transferwise/test-config/test/jest.lint.js & yarn lintless",
-    "test:visual": "yarn chromatic CHROMATIC_PROJECT_TOKEN || true",
+    "test:visual": "build-storybook && percy-storybook --widths=1280",
```
Comment (dozortsev) [inline packages/components/package.json:42]: what is `--widths=1280`?

(why not add comments to PR to make it easier to review for others, or you assume that reviewer should know everything? 😄 )

**Review by dozortsev — COMMENTED** [review]
since this seems like a official solution I think we need document it here 

https://github.com/transferwise/neptune-web/blob/enable-ts-for-components/CONTRIBUTING.md#visual-testing

File: .circleci/config.yml (hunk 1 above)
Comment (andreapiras) [reply .circleci/config.yml:11]: Percy required some browsers spin up and some packages contained in this image

**Review by helendurranttw — APPROVED** [review]

**Pull request merged**